### ISSUE TITLE
Meepo toggle autocast poof just casts poof

### ISSUE THREAD

**Issue by kyxa** [ValveSoftware/Dota2-Gameplay#8099]
### Ability name

Poof

### Description

Toggling autocast just casts poof instead of toggling the autocast.

### Example Match ID (and possibly Timestamp)

demo

**Comment by Neacroe** [comment]
Yeah poof also gets bugged when meepos go through portals. Makes several poof at once and you cant control who what or when, and cant get certain meepos to poof into a group anymore.
